model/article-model.js

```diff
@@ -12,10 +12,34 @@ exports.selectArticleById = (id) => {
     
 }
 
+exports.checkArticleExists = (article_id) => {
+  return db
+  .query('SELECT * FROM articles WHERE article_id = $1', [article_id])
+  .then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status : 404, msg: "Not found"})
+    }
```

Comment:
Here is where you're going to have to check whether the article exists or not :)